Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/booster/BoosterNotificationService.kt

```diff
@@ -17,6 +17,9 @@ class BoosterNotificationService @Inject constructor(
     private val timeStamper: TimeStamper,
     private val personCertificatesSettings: PersonCertificatesSettings,
 ) : DccWalletInfoNotificationService {
+
+    override val notificationSenderType: Int = 0xB00CC
```

Comment:
How did you come with these numbers?

Reply:
it can be integer , it can be 0,1,2 for different notification services , I choose hex integer to avoid repeating them accidentally